EdFi.Buzz.UI/src/Feature/StudentRoster/studentCard.css

```diff
@@ -0,0 +1,22 @@
+.card-max-width{
+    max-width: 400px !important;
+  }
+  .bubble-counter {
+      top: 8.4px;
+      left: 23.8px;
+  }
+  .img-size {
+    max-height: 70px;
+    max-width: 90px;
+  }
+  
+  .image-container img {
+    width: 80%;
+    border-radius: 50%;
+    border: 5px solid lightgrey;
+  }
+  
+  .card-footer {
+    padding:0;
+  }
+  
```

Comment:
Can we put these styles inside a styled component and remove the file entirely? The end goal would be that we would have the one globalStyle.ts for our root styles, buzzTheme.ts for our theme, and all CSS in their components without any CSS files.